js/Round.js

```diff
@@ -36,12 +36,27 @@ class Round {
     });
   }
 
+  setCategoryNames() {
```

Comment:
This is the method I wrote that was the main point of this PR.

Reply:
Can you explain this a little bit? I see what the end goal is just not how you got there.

Reply:
``` setCategoryNames() {
    this.categories = this.categories.map(name => {
      const letters = name.split('');
      let spacedName = letters[0].toUpperCase();
      for (let i = 1; i < letters.length; i++) {
        if (letters[i] === letters[i].toUpperCase()
          && letters[i - 1] !== letters[i - 1].toUpperCase()) {
          spacedName += ' ' + letters[i];
        } else {
          spacedName += letters[i];
        }
      }
      return spacedName;
    });
  }```

So this.categories is an array of strings. The method reassigns the array to a mapped version of itself. In the map, the current string is split into an array of letters. A new let variable called spacedName is declared and assigned the value of the first letter in upper case. Then for the rest of the letters, a conditional is checked. If the letter is already capitalized and the letter before it is not capitalized, then a space plus the capital letter is added to the spacedName variable. Otherwise, the unmodified letter is added. This spacedName with spaces is what is returned.